config/pipelines.yml

```diff
@@ -481,28 +484,43 @@ default: &default
             instrument_slot:
               type: :model
               value: position
+            flowcell_id:
+              type: :model
+              value: flowcell_id
             samples:
               type: :array
-              value: requests
+              value: libraries
```

Comment:
Are these libraries through pools? Just getting my head around the naming conventions and the actual data

Reply:
Yes. Samples is what gets sent to the warehouse in the message. We can't use pool as that does not contain the individual request information. And also yes the libraries are through pools.